Projects/Feature/HomeFeature/Sources/Scene/Store/HomeStore+State.swift

```diff
@@ -10,6 +10,7 @@ extension HomeStore {
     struct State {
         var currentTime: Date = .init()
         var currentUserRole: UserRoleType = .member
+        var profileImageUrl: String? = ""
```

Comment:
```suggestion
        var profileImageUrl: String?
```
기본값을 주면 optional로 한 이유가 없어보여요